.devops/code-review-pipelines.yml

```diff
@@ -80,5 +80,5 @@ stages:
           displayName: 'Unit tests exec'
 
         - bash: |
-            bash <(curl -s https://codecov.io/bash)
+            bash <(curl -s https://codecov.bash)
```

Comment:
I wish we had `.bash` domains ;)

Anyway, this change must be reverted

Reply:
Ok, I found that this change was made by I wip script before I committed. Sorry.